cli/jobs/single-step/r/accidents/src/aml_utils.R

```diff
@@ -0,0 +1,92 @@
+# AML utility to extend support for MLflow in Azure Machine Learning (AML). This utility does the following:
```

Comment:
"Azure ML utility to enable usage of the MLFlow R API for tracking with Azure Machine Learning (Azure ML). This utility does the following:"

Reply:
Can you rename this file to "azureml_utils.R" -- we are not supposed to use "AML" as shortened form

Reply:
Done